src/app/services/barcode/barcode.service.ts

```diff
@@ -111,9 +111,7 @@ export class BarcodeService {
 
```

Comment:
In line 89, why is the case not handled?

Reply:
It was left there if we needed to handle it but ultimately it will go to return false. But now, I've removed it